packages/amplify_core/android/build.gradle

```diff
@@ -43,6 +43,6 @@ android {
 }
 
 dependencies {
-    implementation 'com.amplifyframework:core:1.1.3'
+    implementation 'com.amplifyframework:core:main'
```

Comment:
We should remove this change and upgrade to 1.4.1 like other places. When we do that, we probably have to update all the other plugins to rely on the same version until we can abstract the version to one common place.

Reply:
Sounds good updated all android/build.gradle plugins to use 1.4.1